doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -742,9 +742,9 @@ Versioning is important for long-term research data management where metadata an
 
 |image3|
 
-Once you edit your published dataset a new draft version of this dataset will be created. To publish this new version of your dataset, select the "Publish Dataset" button on the top right side of the page. If you were at version 1 of your dataset, depending on the types of changes you had made, you would be asked to publish your draft as either version 1.1 or version 2.0.
+Once you edit your published dataset, a draft version will be created. To publish this draft version, use the “Publish Dataset” button at the top right side of the page.
```

Comment:
```suggestion
Once you edit your published dataset, a draft version will be created. To publish this draft version, use the "Publish Dataset" button at the top right side of the page.
```

Reply:
Are the only changes related to character encoding weirdness with quotation marks and commas? That's what it looks like to me but just wanna make sure.

And then I click "Commit suggestion"?

Reply:
Hmmm, nvm @pdurbin lol. This has been merged into develop. Not sure what change you were suggesting or if it was included.

Reply:
It looks like my suggested changes didn't make it but they were quite minor, yes, about smart vs. straight quotes.

Clicking "commit suggestion" would have added them. Here are the docs: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

Reply:
Hmm okay. @sekmiller did you see this conversation before merging this into develop? Was there something I could've done to make sure my question was seen before it was merged?